### PR TITLE
Added surfacewater features

### PR DESCRIPTION
**Pull request by jrajak10**

### CODE REVIEW

**Comment by jrajak10** [comment]
#67

File: docs/windturbines.js [hunk 1]
```diff
@@ -127,46 +132,79 @@ map.on('load', async function() {
...
-      let turbineIntersection =[];
-      let intersection;
+      let woodlandIntersection;
       for (let i=0; i<woodlandPolygons.length; i++){
-        intersection = turf.intersect(woodlandPolygons[i], turbineCircle);
-        if(intersection){
-          turbineIntersection.push(intersection)
+        woodlandIntersection = turf.intersect(woodlandPolygons[i], turbineCircle);
+        if(woodlandIntersection){
+          woodlandArea += turf.area(turf.unkinkPolygon(turf.rewind(woodlandIntersection)))/(1600 ** 2);
```
Comment (sebastianovide) [inline docs/windturbines.js]: can you use https://turfjs.org/docs/#convertArea instead

File: docs/windturbines.js [hunk 2]
```diff
@@ -127,46 +132,79 @@ map.on('load', async function() {
...
-      if(area < 10){
-        riskLevel = "Low";
+      let waterIntersection;
+      for (let i=0; i<waterPolygons.length; i++){
+        waterIntersection = turf.intersect(waterPolygons[i], turbineCircle);
+        if(waterIntersection){
+          waterArea += turf.area(waterIntersection)/(1600 ** 2);
+          }
+      }
+      if(woodlandArea < 10){
```
Comment (sebastianovide) [inline docs/windturbines.js]: assign constants to the magic numbers. It will make the code more readable

File: docs/windturbines.js [hunk 3]
```diff
@@ -195,6 +250,7 @@ map.on('load', async function() {
 
     let bounds2TurbineArray = await getFeatures(bounds2, 'Equal', 'DescriptiveTerm', 'Wind Turbine', 'Topography_TopographicArea');
     let bounds2WoodlandArray = await getFeatures(bounds2, 'GreaterThanOrEqual', 'SHAPE_Area', '2500000', 'Zoomstack_Woodland');
+    let bounds2WaterArray = await getFeatures(bounds2, 'GreaterThanOrEqual', 'SHAPE_Area', '100000', 'Zoomstack_Surfacewater');
```
Comment (sebastianovide) [inline docs/windturbines.js]: more magic numbers. You could reuse the constant defined before

File: docs/windturbines.js [hunk 4]
```diff
@@ -80,8 +80,10 @@ map.on('load', async function() {
   let bounds = map.getBounds();
 
   //array for bounds1 features - this will be the arrays with all unique features when the map moves
+  //filters out areas less than 2500000m^2 and 100000m^2 for Woodland and Surfacewater features as they nest in larger areas.
   let uniqueTurbineArray = await getFeatures(bounds, 'Equal', 'DescriptiveTerm', 'Wind Turbine', 'Topography_TopographicArea');
   let uniqueWoodlandArray = await getFeatures(bounds, 'GreaterThanOrEqual', 'SHAPE_Area', '2500000', 'Zoomstack_Woodland');
+  let uniqueWaterArray = await getFeatures(bounds, 'GreaterThanOrEqual', 'SHAPE_Area', '100000', 'Zoomstack_Surfacewater');
```
Comment (sebastianovide) [inline docs/windturbines.js]: magic number

File: docs/windturbines.js [hunk 5]
```diff
@@ -127,46 +135,82 @@ map.on('load', async function() {
...
+      let waterIntersection;
+      for (let i=0; i<waterPolygons.length; i++){
+        waterIntersection = turf.intersect(waterPolygons[i], turbineCircle);
+        if(waterIntersection){
+          waterArea += turf.area(waterIntersection);
+          waterAreaMiles2 = turf.convertArea(waterArea, 'metres', 'miles')
+          }
+          
+      }
+      if(woodlandAreaMiles2 < 10){
```
Comment (sebastianovide) [inline docs/windturbines.js]: magic number

File: docs/windturbines.js [hunk 6]
```diff
@@ -127,46 +135,82 @@ map.on('load', async function() {
...
+        riskScore += 2;
+      }
+
+      if(riskScore < 2){
+        overallRisk = "Low";
         element.style.backgroundImage = "url('windturbineicongreen.png')";
       }
-      else if(area >=10 && area <35){
-        riskLevel = "Medium";
+      else if(riskScore >=2 && riskScore < 3){
```
Comment (sebastianovide) [inline docs/windturbines.js]: more magic numnber

**Pull request merged**